app/handlers/handlers_connectors.py

```diff
@@ -54,7 +54,12 @@ def get_connector_pod(
                 "imagePullPolicy": "Always",
                 "name": "connector",
                 "securityContext": {
-                    "allowPrivilegeEscalation": False
+                    "allowPrivilegeEscalation": False,
+                    "capabilities": {
+                        "drop": ["ALL"],
+                    },
+                    "runAsNonRoot": True,
+                    "runAsUser": 65532,
```

Comment:
We should change this to 1000 since we have an appuser account that the container runs as

```
appuser:x:1000:1000::/home/appuser:/bin/sh
```

Reply:
You are right 🤦‍♂️

Reply:
This image is the **connector** image (distroless, with `nonroot` user whos id is `65532`)